terraform.rb

```diff
@@ -52,7 +52,7 @@ def install
     end
 
     cd terrapath do
-      # v0.6.12 - source contains tests which fail if these environment variables are set locally.
+      # v0.6.13 - source contains tests which fail if these environment variables are set locally.
```

Comment:
Comments with version numbers, I think it should stay the same, as that should denote when this error started (unless we've fixed it).